iocage_lib/ioc_start.py

```diff
@@ -951,21 +951,22 @@ def start_network(self, vnet, nat=False):
                     net_configs
                 ):
                     try:
-                        iocage_lib.ioc_common.checkoutput(
-                            [
-                                'setfib', self.exec_fib, 'jexec',
-                                f'ioc-{self.uuid}',
-                                'route'
-                            ] + list(
-                                filter(
-                                    bool, [
-                                        'add', '-6' if ipv6 else '',
-                                        'default', default_route
-                                    ]
-                                )
-                            ),
-                            stderr=su.STDOUT
-                        )
+                        if default_route:
```

Comment:
@joachimmathes i suggest we donot do the indentation but rather take care of this in the filter at L950. There changing it to `lambda v: v[0] != 'none' and v[1] != 'none'` should be sufficient to not configure a default route for ipv6/4 if not specified. Also your fix would still fail, `default_route` by default is `none`, so the condition is true.

Reply:
I've implemented your solution.